engines/terraform/terraform.go

```diff
@@ -59,7 +59,16 @@ func (e *TerraformEngine) GetPluginManifestsForType(typ string) (map[string]*Res
 }
 
 // Apply the engine to the target environment
-func (e *TerraformEngine) Apply(appSpec *app_spec_schema.Application) (string, error) {
+func (e *TerraformEngine) Apply(appSpec *app_spec_schema.Application) (result string, err error) {
+	// Defer panic recovery to gracefully handle any panics
```

Comment:
I think you should add a longer comment here about how and JSII errors will result in a panic, which is why we've needed to add this.

Reply:
done